omegatrade-app/frontend/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="header header-fixed">
+    <div class="header-inner">
+        <div class="logo-section">
+                <a href="javascript:void(0)"><img class="logo" src="assets/images/to_crop_2 (1).png" alt=""></a>
+        </div>
+        <div class="profile-pic" *ngIf="this.user">
+            <div mat-icon-button [matMenuTriggerFor]="menu">
+                <img *ngIf="this.user.photoUrl" [src]="this.user.photoUrl" alt="">
+                <span *ngIf="!this.user.photoUrl" class="material-icons" style="font-size: 40px;padding-right: 10px;cursor:pointer">account_circle</span>
+            </div>
+              <mat-menu #menu="matMenu">
+                <span mat-menu-item>
+                  <mat-icon>email</mat-icon>
+                  <span>{{ this.user.businessEmail }}</span>
+                </span>
+                <span mat-menu-item >
+                  <mat-icon>account_circle</mat-icon>
+                  <span>{{ this.user.fullName }}</span>
+                </span>
+                <span mat-menu-item (click)="this.logOut()">
+                  <mat-icon>login</mat-icon>
```

Comment:
`login` icon for logout action seems strange.

Reply:
The above issue has been resolved